datalad/local/add_readme.py

```diff
@@ -101,12 +101,17 @@ def __call__(filename='README.md',
                 dataset.save(
                     path='.gitattributes',
                     message="[DATALAD] Configure README to be in Git",
-                    to_git=True
+                    to_git=True,
+                    return_type='generator',
+                    result_renderer='disabled'
```

Comment:
needs to be wrapped into `yield from` or would stop doing anything

Reply:
oh thanks for catching this! it is a bit worrisome that the tests did not pick this up...